internal/pkg/stream/ecs.go

```diff
@@ -5,6 +5,7 @@ package stream
 
 import (
 	"fmt"
+	"github.com/aws/copilot-cli/internal/pkg/aws/cloudwatch"
```

Comment:
nit: same with the imports here, it's weird that the IDE doesn't do this properly 💭